eliot-service/eliot/cache_manager.py

```diff
@@ -26,10 +26,10 @@
 import sys
 import traceback
 
+from boltons.dictutils import OneToOne
```

Comment:
boltons has a `OneToOne` map which simplifies keeping track of each direction of the map. There are some other things in boltons that I think I want to use, so I decided it's worth pulling it in as a dependency now.